packages/react/src/primitives/Tabs/TabsItem.tsx

```diff
@@ -17,23 +17,24 @@ const TabsItemPrimitive: Primitive<TabsItemProps, 'button'> = (
   { className, value, children, onClick, as = 'button', role = 'tab', ...rest },
   ref
 ) => {
-  const { activeTab, setActiveTab } = React.useContext(TabsContext);
+  const { activeTab, setActiveTab, groupId, whitespaceValue } =
+    React.useContext(TabsContext);
```

Comment:
Since `whitespaceValue` is a static value, there is no need to pass it through context as it will not update